SNIPS/snip-23.md

```diff
@@ -0,0 +1,110 @@
+---
+snip: 23
```

Comment:
Please change to 24. Discussion on numbering policy is ongoing [here](https://community.starknet.io/t/snip-numbering-policy-and-draft-snips/115023)